dlp/snippets/jobs/jobs_test.go

```diff
@@ -34,25 +34,25 @@ func setupPubSub(projectID, topic, sub string) (*pubsub.Subscription, error) {
 	ctx := context.Background()
 	client, err := pubsub.NewClient(ctx, projectID)
 	if err != nil {
-		return nil, fmt.Errorf("pubsub.NewClient: %w", err)
+		return nil, fmt.Errorf("pubsub.NewClient: %v", err)
 	}
 	// Create the Topic if it doesn't exist.
 	t := client.Topic(topic)
 	if exists, err := t.Exists(ctx); err != nil {
-		return nil, fmt.Errorf("error checking PubSub topic: %w", err)
+		return nil, fmt.Errorf("error checking PubSub topic: %v", err)
 	} else if !exists {
 		if t, err = client.CreateTopic(ctx, topic); err != nil {
-			return nil, fmt.Errorf("error creating PubSub topic: %w", err)
+			return nil, fmt.Errorf("error creating PubSub topic: %v", err)
```

Comment:
Revert. `%w` is the correct formatting directive.

Reply:
fixed it.